app/controllers/physical_ticket_controller.rb

```diff
@@ -13,6 +13,8 @@ def show
     @file_name = "ticket_for_#{@conference.short_title}"
     @user = @physical_ticket.user
     @ticket_layout = @conference.ticket_layout.to_sym
+    @qrcode = RQRCode::QRCode.new(@physical_ticket.token)
```

Comment:
Is this instance variable needed? Is not consumed by any of the views, isn't it?
I prefer this to be a simple local variable (or just replace this code in the next line because isn't used anymore).